openapi/openapi.yaml

```
@@ -1120,7 +1124,7 @@ paths:
                     base:
                       type: string
                       description: Base image to build stack upon.
-                      example: quay.io/thoth-station/s2i-thoth-ubi8-py36
+                      example: quay.io/thoth-station/s2i-thoth-ubi8-py38:v0.32.3
```

```suggestion
                      example: quay.io/thoth-station/s2i-thoth-ubi8-py38:latest
```

lets keep this latest, so we wont have to update this schema with each release

---

Added in https://github.com/thoth-station/management-api/pull/868/commits/bc244f3dfd741b07ed0e6be47796d36246e706fc, thanks!